Command/DeleteOrphanIndexesCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace EMS\CoreBundle\Command;
+
+use Elasticsearch\Client;
+use EMS\CoreBundle\Service\AliasService;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Elasticsearch\Common\Exceptions\Missing404Exception;
+
+class DeleteOrphanIndexesCommand extends EmsCommand
+{
+    /** @var LoggerInterface */
+    protected $logger;
+    /** @var Client */
+    protected $client;
+    /** @var AliasService */
+    protected $aliasService;
+
+    public function __construct(LoggerInterface $logger, Client $client, AliasService $aliasService)
+    {
+        $this->logger = $logger;
+        $this->client = $client;
+        $this->aliasService = $aliasService;
+        parent::__construct($logger, $client);
+    }
+
+    protected function configure()
+    {
```

Comment:
assigning command name is now done using a variable:

https://symfony.com/doc/current/console.html
protected static $defaultName = 'app:create-user';

Reply:
But do not do this when extending from EMSCommand ! because it will get overriden (to be reviewed in a later stade)

Reply:
How do you mean? DeleteOrphanIndexesCommand will overwrite EMSCommand anyway won't it?

Reply:
protected static $defaultName = 'app:create-user'; Would not work because setName is called in the parent. So setting a defaultName in this extended class would not work I think (to be tested?)

Reply:
This works, instances of classes always use their own properties instead of parent properties, even when using parent methods. I also tested this to make sure.